lms/djangoapps/commerce/utils.py

```diff
@@ -327,9 +327,13 @@ def _send_refund_notification(user, refund_ids):
 
     tags = ['auto_refund']
 
-    if theming_helpers.is_request_in_themed_site():
-        # this is not presently supported with the external service.
-        raise NotImplementedError("Unable to send refund processing emails to support teams.")
+    # These changes are only for UCSD to send refund failure notification on unenroll
```

Comment:
I think it would be better if we add some kind of prefix (such as `UCSD_CUSTOM` for such comments and changes that are specific to UCSD or any other enterprise. In this way it would be easier to search for such changes in the future.

Reply:
it's a nice suggestion I guess (y)